### DISCUSSION TITLE
Lazer to stable migration

### DISCUSSION THREAD

**Discussion by PetHedgehog** [ppy/osu#25861] [Ideas]
I'm sure many asked for this and i cant find anywhere talking about a reason why this is not a thing but please allow us to export all our downloaded beatmaps and local scores to stable, the latest update really made me think ill never play this ever again as i don't understand and wasn't given an explanation why classic mod was made unviable, i can not play lazer without it and I'm sure many cant either but we are stuck on lazer and cant move to stable for no logical reason. Or at least provide us with a serious answer why are we not allowed to go back.

**Comment by peppy** [comment]
This is out of scope of this project, but there's third party projects which allow you to do this. Please search on google or github.

Explanation was given in the chagelog and video, please read / watch (although it sounds like you've made up your mind).

**Reply by PetHedgehog** [reply]
could you point me to direction of such tools, after this update i seriously dont know what to do anymore, i had searched before for such a thing after some other change i couldnt handle but i wasnt successfull

**Reply by PetHedgehog** [reply]
ive found one tool yet it doesnt work for the update, also i was asking for an explanation why lazer itself doesnt allow us to transfer back and why do we have to use third party toolsx
